app/serializers/application_serializer.rb

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+
+class ApplicationSerializer
+  include JSONAPI::Serializer
```

Comment:
@Tuubz Do you think we should have concerns for the serializer and moving render method for success and error inside? 🤔 If so I will refactor on PR #18

Reply:
Sounds good to me 👍

Reply:
Guys, stop moving everything into concerns. It makes the code harder to read and discover ⚠️ Only use concerns for complex logic that needs to be shared. Defining methods on the application controller would work better for rendering logic (i.e, there is nothing complex)